### PR TITLE
Feat/use black lint

### LINKED ISSUE #619: Change code formatter to Black from YAPF
Homogenize code style with rest of Clinical Genomics which is using Black: https://github.com/psf/black and remove YAPF

### PR DESCRIPTION
**Pull request by hassanfa**
### This PR:

Resolves :
- Uses Black as linter instead of YAPF Fixes #619 

### Review and tests: 
- [ ] Tests pass
- [ ] Code review
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

**Comment by hassanfa** [comment]
I removed container workflows temporarily...

File: .github/workflows/black_linter.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Lint
```
Comment (ashwini06) [inline .github/workflows/black_linter.yml:1]:
```suggestion
name: black_linter
```

File: BALSAMIC/conda/balsamic.yaml [hunk 2]
```diff
@@ -4,6 +4,6 @@ channels:
   - defaults
 
 dependencies:
-  - python=3.6 
-  - pip=9.0.3
+  - python=3.7.9
```
Comment (ashwini06) [inline BALSAMIC/conda/balsamic.yaml:7]:
```suggestion
  - python=3.7
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -47,6 +47,7 @@ Changed:
 * Split Tumor-Normal ``cnvkit batch`` into individual commands
 * Moved params that are used in multiple rules to constants #711
 * Changed the way conda and non-conda bioinfo tools version are parsed
+* Python code formatter changed from Black to YAPF
```
Comment (ashwini06) [inline CHANGELOG.rst:50]:
```suggestion
* Python code formatter changed from Black to YAPF #619 
```

**Review by ashwini06 — APPROVED** [review]
Good job 💯 jus minor comments

File: BALSAMIC/conda/balsamic.yaml [hunk 4]
```diff
@@ -4,6 +4,6 @@ channels:
   - defaults
 
 dependencies:
-  - python=3.6 
-  - pip=9.0.3
+  - python=3.7.9 
+  - pip
```
Comment (khurrammaqbool) [inline BALSAMIC/conda/balsamic.yaml:8]: Although black won't complain, we should encourage use of channels and versions with all conda packages.

**Review by khurrammaqbool — APPROVED** [review]
The code looks great with new formatting. Great work! 💯 👍

**Pull request merged**